### PR TITLE
frontend/fix:#4004 Notification sound issue fix PROD

### PR DESCRIPTION
**Pull request by MercyQueen**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

notification issue fix.

### CODE REVIEW

**Review by aryan-13 — APPROVED** [review]

File: Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/NotificationUtils.java [hunk 1]
```diff
@@ -387,7 +387,6 @@ public static void showNotification(Context context, String title, String msg, J
                 
         if (TRIP_STARTED.equals(notificationType)) {
             if (key.equals("USER")) {
-                startMediaPlayer(context,R.raw.notify_otp_sound, true);
```
Comment (witcher-shailesh) [inline Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/NotificationUtils.java:390]: why is this removed?

Comment (Vignesh-772) [reply Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/NotificationUtils.java:390]: This is not required. normal notification sound itself is working fine.

Comment (witcher-shailesh) [reply Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/NotificationUtils.java:390]: what kind of sound notify_otp play?

**Review by aryan-13 — APPROVED** [review]

File: Frontend/android-native/app/src/user/nammaYatri/assets/juspay/index_bundle.js
Comment (Vignesh-772) [inline Frontend/android-native/app/src/user/nammaYatri/assets/juspay/index_bundle.js:1]: can you add this file back?

**Review by witcher-shailesh — APPROVED** [review]
